Source/WebKit/Shared/WebHitTestResultData.cpp

```diff
@@ -101,53 +113,19 @@ WebHitTestResultData::WebHitTestResultData(const HitTestResult& hitTestResult, c
     , isDownloadableMedia(hitTestResult.isDownloadableMedia())
     , mediaIsInFullscreen(hitTestResult.mediaIsInFullscreen())
     , isActivePDFAnnotation(false)
-    , elementType(ElementType::None)
+    , elementType(elementTypeFromHitTestResult(hitTestResult))
```

Comment:
To minimize breaking other things in this PR, let's limit the changes here to just adding an optional response.  This did look wrong before, but let's fix it in a separate PR.